src/R/Actions/Impl/Utility/RInstallation.cs

```diff
@@ -259,7 +259,7 @@ public static class RInstallation {
         }
 
         public static Version GetRVersionFromFolderName(string folderName) {
-            if (folderName.StartsWith("R-")) {
+            if (folderName.StartsWith("R-", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
This code is used in setup custom actions via linked items and custom actions can't reference assemblies that haven't been installed yet :-)